sharktank/sharktank/examples/sharding/export_ffn_net.py

```diff
@@ -30,16 +30,16 @@ def create_theta(
     gate_weight = torch.rand(hidden_dim, primary_dim, dtype=torch.float16)
     up_weight = torch.rand(hidden_dim, primary_dim, dtype=torch.float16)
     sharded_gate_weight = SplitPrimitiveTensor(
-        name="ffn_gate.weight", shard_dim=0, ts=gate_weight.split(split_size, dim=0)
+        name="ffn_gate.weight", shard_dim=0, ts=gate_weight.split(split_size, dim=0), devices=tuple(range(shard_count)), devices_pinned=True
```

Comment:
Lets avoid setting device placement information on non pipeline parallel programs. Overall we should be able to default safely without changing things.

Something to consider is that authors for a tensor-parallel program should not need to think about the pipeline parallel behavior. Same as unsharded programs should be able to not think about sharding.